### DISCUSSION TITLE
Our name!

### DISCUSSION THREAD

**Discussion by denempath** [Open-Archypelago/Archypelago#13] [Polls]
I like the idea of "tractor boys" but it is not forward looking because it is not gender neutral. I asked google gemini for some alternatives...

**Comment by denempath** [comment]
@Jandwala , @DavidKilburn - please add your votes to the [our name poll](https://github.com/orgs/Open-Archypelago/discussions/13#discussion-6584868)

**Reply by DavidKilburn** [reply]
Tractor Boys / Crew sounds like a boy band - and we are way too old for modern boy band references, even if it is "hip" these days.  The suggestions are all pretty questionable in my book, the only one which has legs in my opinion is "ground breakers" - although we don't want to get mixed up in the "construction" or "demolition" space, eg. building roads!  What about "terra firma"?

**Reply by denempath** [reply]
we have a winner - "Tiller Tillers"
